src/Details.js

```diff
@@ -88,4 +82,6 @@ Details.propTypes = {
   ...sx.propTypes
 }
 
+Details.Context = DetailsContext
```

Comment:
Maybe we should expose a `useDetails` hook instead of the entire context object? Because technically this means that `Details.Context.Consumer` and `Details.Context.Provider` are also part of the `Details` public API, and I don't think we want people rolling their own Details provider.

Reply:
done!